server/src/domain/entities/transaction.rs

```diff
@@ -0,0 +1,94 @@
+use serde::{Deserialize, Serialize};
+use std::fmt;
+use utoipa::ToSchema;
+
+/// Статусы транзакции.
+#[derive(Debug, PartialEq, Clone, Deserialize, Serialize, ToSchema)]
+pub enum Operation {
+    Registration,
+    Replenish,
+    Withdraw,
+    TransferIncrease,
+    TransferDecrease,
+}
+
+/// to_string().to_lowercase() для значения в Json
+impl fmt::Display for Operation {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
+/// Структура транзакции.
+#[derive(Debug, Default, Clone, Deserialize, Serialize, ToSchema)]
+pub struct Transaction {
+    pub id: u32,
+    pub operation: String,
```

Comment:
```suggestion
    pub operation: Operation,
```

Может, это так подразумевалось?

Reply:
Да, все верно!
Поясню. `String` поставил, потому, что в json обычно `lowercase`. `Operation` будет с большой буквы)
Просто хотел потренироваться в реализации `Display`, не более)

Reply:
Можно использовать [`#[serde(rename_all(...))]`](https://serde.rs/variant-attrs.html#rename_all)

Reply:
интересно, поизучаю!
пока сделаю как ты предложил. 
`pub operation: Operation`